packages/playwright-core/src/server/trace/viewer/traceViewer.ts

```diff
@@ -26,7 +26,7 @@ import { serverSideCallMetadata } from '../../instrumentation';
 import { createPlaywright } from '../../playwright';
 import { ProgressController } from '../../progress';
 
-export async function showTraceViewer(traceUrls: string[], browserName: string, headless = false, preferredPort?: number): Promise<BrowserContext | undefined> {
+export async function showTraceViewer(traceUrls: string[], browserName: string, headless = false, host?: string, preferredPort?: number): Promise<BrowserContext | undefined> {
```

Comment:
Let's turn this into the options: { headless?: boolean, host?: string, port?: number }

Reply:
Options object is now used for `headless`, `host` and `port`